lib/hotel_rooms.rb

```diff
@@ -0,0 +1,37 @@
+require 'date'
+
+module Hotel
+  class HotelRooms
+    attr_reader :rooms, :reservations
+
+    def initialize
+      @rooms = [:rooms]
+      # fill in hotel rooms
+      @reservations = {}
+      set_up_rooms #calling this method
+      set_up_reservations #calling this method
+    end
+
+
+  def set_up_rooms
+    #creates list of hotel rooms
+    19.times do |room_number|
+      data = {}
+      data[:room_number] = (room_number + 1)
+      @rooms << Hotel::Room.new(data)
+    end
+      return @rooms
+  end
+
+  def list_rooms #outputs list of rooms
+    return @rooms
+  end
+
+  def set_up_reservations #sets reservation's key to room #s
+    @reservations = {}
+    @reservations.each do |key, value|
```

Comment:
What is this method trying to do?  Why have a hash of reservations and why have each element point to a list of all the rooms.